bot/plotting/random_plot_generator.py

```diff
@@ -182,35 +187,35 @@ def random_plot_generator(
                         solver=solver
                     )
                     generate_summary_variables(solution)
-                    return (
-                        model,
-                        parameter_values,
-                        None,
-                        chemistry,
-                        solver,
-                        True,
-                        cycle_received,
-                        number,
-                        False,
-                    )
+                    return_dict["model"] = model
```

Comment:
It is totally OK this way, but it might look cleaner if you use `return_dict.update()` instead.

Reply:
(same applies to the other cases)

Reply:
Updated in #10, thanks!